docs/README.md

```diff
@@ -6,9 +6,8 @@
 
 # Components
 
-* [Psl\Async](../src/Psl/Async/README.md)
-* [Psl\Default](../src/Psl/Default/README.md)
```

Comment:
this removed default which is now in next?

Reply:
nice catch!